src/app/components/profile/profile.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7182177)
```suggestion
import { Component, OnInit } from "@angular/core";
```